invenio_files_rest/views.py

```diff
@@ -101,7 +104,7 @@ def invalid_subresource_validator(value):
         load_from='Content-MD5',
         location='headers',
     ),
-    'x_invenio_tags': fields.Str(
+    'x_invenio_tags': QueryString(
```

Comment:
I think it would make more sense if we call it something like `x-invenio-files-tagging`, just to add a valid namespace to the header key.